### PR TITLE
Add man pages for ocitools

### PR DESCRIPTION
**Pull request by rhatdan**
Also fix up Makefile so that it will install ocitools and man pages.

Signed-off-by: Dan Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Replaces https://github.com/opencontainers/ocitools/pull/30

File: man/ocitools-bvalidate.1.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# NAME
+ocitools-bvalidate - Validate a OCI bundle
+
+# SYNOPSIS
+**ocitools bvalidate**
+[**--path**[=*PATH*]
+
+# DESCRIPTION
+
+Valida an OCI bundle
```
Comment (mrunalp) [inline man/ocitools-bvalidate.1.md]: typo: Validate

File: man/ocitools-generate.1.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+# DESCRIPTION
+
+Generate a process in a new container. **ocitools generate** starts a process with its own
+file system, its own networking, and its own isolated process tree. The IMAGE
+which starts the process may define defaults related to the process that will be
+generate in the container, the networking to expose, and more, but **ocitools generate**
+gives final control to the operator or administrator who starts the container
+from the image. For that reason **ocitools generate** has more options than any other
+Oci command.
```
Comment (mrunalp) [inline man/ocitools-generate.1.md]: s/Oci/OCI

File: man/ocitools-generate.1.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+  Use ipc namespace
+
+**--network**
+  Use network namespace
+
+**--no-new-privileges**
+  Set no new privileges bit for the container process.  Setting this flag
+  will block the container processes from gaining any additonal privileges
+  using tools like setuid apps.  It is a good idea to run unprivileged
+  contaiers with this flag.
```
Comment (mrunalp) [inline man/ocitools-generate.1.md]: typo: containers

File: Makefile [hunk 4]
```diff
@@ -1,12 +1,24 @@
...
 	go build -tags "$(BUILDTAGS)" -o ocitools .
 	go build -tags "$(BUILDTAGS)" -o runtimetest ./cmd/runtimetest
 
-install:
-	cp ocitools /usr/local/bin/ocitools
+.PHONY: man
+man:
+	go-md2man -in "man/ocitools.1.md" -out "ocitools.1"
+	go-md2man -in "man/ocitools-generate.1.md" -out "ocitools-generate.1"
+	go-md2man -in "man/ocitools-bvalidate.1.md" -out "ocitools-bvalidate.1"
```
Comment (mrunalp) [inline Makefile]: bvalidate -> validate

File: man/ocitools-generate.1.md [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+**--cap-add**=[]
+   Add Linux capabilities
+
+**--cap-drop**=[]
+   Drop Linux capabilities
+
+**--cwd**=PATH
+   Current working directory for the process
+
+**-e**, **--env**=[]
```
Comment (mrunalp) [inline man/ocitools-generate.1.md]: We don't have -e short option.

File: man/ocitools-generate.1.md [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+  containers with this flag.
+
+**--mount**
+  Use a mount namespace
+
+**--mount-cgroups**
+  Mount cgroups (rw,ro,no)
+
+**--os**=OS
+  Operating used within the container
```
Comment (mrunalp) [inline man/ocitools-generate.1.md]: Operating system

File: man/ocitools-generate.1.md [hunk 7]
```diff
@@ -0,0 +1,283 @@
...
+[**--selinux-label**[=*[]*]]
+[**--sysctl**[=*[]*]]
+[**--tmpfs**[=*[]*]]
+[**--uid**[=*[]*]]
+[**--uidmappings**[=*[]*]]
+[**--uts**]
+[ARG...]
+
+# DESCRIPTION
+oci generate is used to generate a config.json (oci Spec file) to be used to
```
Comment (mrunalp) [inline man/ocitools-generate.1.md]: s/oci generate/ocitools generate/

Comment (mrunalp) [inline man/ocitools-generate.1.md]: s/Spec/spec

File: man/ocitools-generate.1.md [hunk 8]
```diff
@@ -0,0 +1,283 @@
...
+instantiate an OCI container.  This config.json file can be placed into a
```
Comment (mrunalp) [inline man/ocitools-generate.1.md]: s/OCI/oci/ or vice versa in the paragraph.

File: man/ocitools-generate.1.md [hunk 9]
```diff
@@ -0,0 +1,283 @@
...
+# OPTIONS
+**--apparmor**="PROFILE"
+   Specifies the the apparmor profile for the container
+
+**--arch**="ARCH"
+   Architecture used within the container.
+   "amd64"
+
+**--args**=OPTION
+   Command to run in the container
```
Comment (mrunalp) [inline man/ocitools-generate.1.md]: Can add a line to say that it can be specified multiple times to add arguments.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**